betree/pmdk/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "pmdk"
+version = "0.1.0"
+edition = "2021"
+build = "build.rs"
+links = "pmem"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
We should probably add ourselves as authors, just in case :D.
There is a different `pmdk` crate on crates.io and people may confuse it with ours.